### PR TITLE
Add FAQ: "How do I get started using the Team feature?"

### PR DESCRIPTION
**Pull request by cbrownstein-lambda**

### CODE REVIEW

File: content/en/cloud/get-started-teams/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+   {{% alert title="Warning" color="warning" %}}
+   **Be sure to invite only trusted persons to your team!**
+
+   Currently, the only differences between the _Admin_ and _Member_ roles are
+   that an _Admin_ can:
+
+   - Invite others to the team.
+   - Delete team accounts.
+   - Modify payment information.
+   - Change the team name.
```
Comment (cbrownstein-lambda) [inline content/en/cloud/get-started-teams/index.md]: @shawnrusaw-lambdalabs Is it accurate that these are the only differences between the _Admin_ and _Member_ roles?

Comment (shawnrusaw-lambdalabs) [reply content/en/cloud/get-started-teams/index.md]: - remove members

Comment (cbrownstein-lambda) [reply content/en/cloud/get-started-teams/index.md]: What are your thoughts on the suggested change, below?

Teammates with an _Admin_ role can remove other teammates with an _Admin_ role—not just teammates with a  _Member_ role.

```suggestion
   Currently, the only differences between the _Admin_ and _Member_ roles are
   that an _Admin_ can:

   - Invite others to the team.
   - Remove others from the team.
   - Modify payment information.
   - Change the team name.
```

Comment (shawnrusaw-lambdalabs) [reply content/en/cloud/get-started-teams/index.md]: i'm fine with that

**Pull request merged**